data_handler.py

```diff
@@ -21,6 +21,9 @@ class BaseDataHandler:
     def __init__(self, *args, **kwargs):
         self._sample_size = kwargs.pop('sample', 0)
         self._skip_scrape = kwargs.pop('no_scrape', False)
+        self._outputter = kwargs.pop('outputter')
```

Comment:
Consider setting a default (probably `False`) with the changes for the other comment

Reply:
I would disagree with this, adding a default here would only hide errors in other code. Potentially causing unwanted behavior due to using an unexpected default, requiring debugging to figure out why it does something wrong. Where not having a default it would fail faster and tell you that it where to look for the error.

Reply:
With the other change this makes more sense to do so i changed this aswell.